### PR TITLE
Added a missing apostrophe

### PR DESCRIPTION
**Pull request by amkhrjee**
> Issue number to be fixed (ie 'Fixes #123', if there is an associated issue)

> What was wrong/why is this fix needed? (quick summary only)
The word 'TCP' in the second paragraph was missing an apostrophe. 

> Anything else that could help us review it

### CODE REVIEW

**Review by Ryuno-Ki — APPROVED** [review]
Looks good to me.

File: files/en-us/glossary/tcp/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 ---
 <p id="Summary"><strong>TCP (Transmission Control Protocol)</strong> is an important network {{Glossary("protocol")}} that lets two hosts connect and exchange data streams.  TCP guarantees the delivery of data and packets in the same order as they were sent.  Vint Cerf and Bob Kahn, who were DARPA scientists at the time, designed TCP in the 1970s.</p>
 
-<p>TCP role is to ensure the packets are reliably delivered, error free.  TCP has concurrence control, which means the initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.</p>
+<p>TCP's role is to ensure the packets are reliably delivered, error free.  TCP has concurrence control, which means the initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/glossary/tcp/index.html:14]:
```suggestion
<p>TCP's role is to ensure the packets are reliably delivered, error free. TCP has concurrence control, which means the initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.</p>
```

I think, an extra space sneaked in here, too.

**Comment by Ryuno-Ki** [comment]
Thank you and congratulations on getting your first PR merged! :partying_face: Welcome aboard

**Comment by SVarvello** [comment]
This is a hacker using SVsrvello not me doing it

**Comment by SVarvello** [comment]
Google





Scott Varvello


Scott Varvello

I shared my profile with you






View on Google Maps




Google

© 2018 Google LLC 1600 Amphitheatre Parkway, Mountain View, CA 94043

Sent by Scott Varvello via Google Maps

**Pull request merged**